modules/BookDisplay.js

```diff
@@ -0,0 +1,30 @@
+import { saveDataToLocalStorage } from './LocalStorage.js';
+
+export default function showBooks(books) {
+  function addElement(elementType, parent, className) {
+    const element = document.createElement(elementType);
+    element.classList.add(className);
+    parent.appendChild(element);
+    return element;
+  }
+
```

Comment:
- [ ] This milestone encourages us to use E6 JavaScript coding standards. Kindly use an arrow function here